owesome.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-sequences */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-plusplus */
+/* eslint-disable linebreak-style */
+/* eslint-disable indent */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable linebreak-style */
+/* eslint-disable eqeqeq */
+/* eslint-disable consistent-return */
+/* eslint-disable linebreak-style */
+/* eslint-disable array-callback-return */
+/* eslint-disable no-unused-expressions */
```

Comment:
- [ ] [OPTIONAL] It is not appropriate to disable such amount of linter rules because they provide vast room for syntax errors. Please make sure that you don't do it a lot in your next projects 😉